src/.nuget/packages.builds

```diff
@@ -1,10 +1,6 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Can we change this to an SDK style project to include Directory.Build.props automatically. Importing it will happen much later and will not play as nicely with Arcade.